### DISCUSSION TITLE
Github Organization Verification for creating New Organisation is not working

### DISCUSSION THREAD

**Discussion by X-itachi-X** [community/community#57335] [Other Feature Feedback, Questions, & Ideas]
Why is New organisation creation is not working whenever I fill the details and try to create the captcha panel start to load and show green tick but after submitting it shows captcha answer not right and it reopens that page.

**Comment by ImaniAN** [comment]
could be an HTTP request/response issue with captcha side of things... 

try deleting all cookies and site data for Github on your browser, resign into GitHub, and do the organization creation process again and see if that works... 

Good luck

**Comment by DSDmark** [comment]
Hi @X-itachi-X,

Not sure. But, lots of people facing same captcha issue like this discussion #57314. Now is working fine. So, try again. It may work.

**Reply by X-itachi-X** [reply]
yeah its working fine now.

**Reply by ImaniAN** [reply]
Please close the discussion
